docs/examples/bifacial/plot_bifi_model_pvwatts.py

```diff
@@ -0,0 +1,111 @@
+"""
+Bifacial Modeling - procedural
+==============================
+
+Example of bifacial modeling using procedural method
+"""
+
+# %%
+# This example shows how to complete a bifacial modeling example using the
+# :py:func:`pvlib.pvsystem.pvwatts_dc` with the
+# :py:func:`pvlib.bifacial.pvfactors_timeseries` function to transpose
+# GHI data to both front and rear Plane of Array (POA) irradiance.
+
+import pandas as pd
+from pvlib import location
+from pvlib import tracking
+from pvlib import bifacial
+from pvlib import temperature
+from pvlib import pvsystem
+import matplotlib.pyplot as plt
+
+# using Greensboro, NC for this example
+lat, lon = 36.084, -79.817
+tz = 'Etc/GMT+5'
+times = pd.date_range('2021-06-21', '2021-06-22', freq='1T', tz=tz)
+
+# create location object and get clearsky data
+site_location = location.Location(lat, lon, tz=tz, name='Greensboro, NC')
+cs = site_location.get_clearsky(times)
+
+# get solar position data
+solar_position = site_location.get_solarposition(times)
+
+# set ground coverage ratio and max_angle to
+# pull orientation data for a single-axis tracker
+gcr = 0.35
+max_phi = 60
+orientation = tracking.singleaxis(solar_position['apparent_zenith'],
+                                  solar_position['azimuth'],
+                                  max_angle=max_phi,
+                                  backtrack=True,
+                                  gcr=gcr
+                                  )
+
+# set axis_azimuth, albedo, pvrow width and height, and use
+# the pvfactors  engine for both front and rear-side absorbed irradiance
+axis_azimuth = 180
+pvrow_height = 3
+pvrow_width = 4
+albedo = 0.2
+irrad = bifacial.pvfactors_timeseries(solar_position['azimuth'],
```

Comment:
Same comment about suppressing warnings, although I do not understand why only the RuntimeWarnings are raised here.

Reply:
I think this is because the Shapely warnings happen when python imports pvfactors, not because of any calculations.  That, plus the fact that all gallery examples run in the same python process (different namespaces but shared `sys.modules`), means that pvfactors is only actually imported once during the docs build, so any warning emitted during the import will only appear in whatever example happens to import it first.  

+1 to suppressing the Shapely warnings, and I wouldn't object to suppressing the runtime warnings as well.